ivadomed/transforms.py

```diff
@@ -122,11 +123,15 @@ def __call__(self, sample):
 
         if self.labeled:
             gt_data = sample['gt']
-            rdict['gt'] = self.resample_bin(gt_data, wshape_new, hshape_new)
+            rdict['gt'] = []
+            for gt in gt_data:
+                rdict['gt'].append(self.resample_bin(gt, wshape_new, hshape_new))
 
         if sample['roi'] is not None:
             roi_data = sample['roi']
-            rdict['roi'] = self.resample_bin(roi_data, wshape_new, hshape_new, 0.0)
+            rdict['roi'] = []
+            for roi in roi_data:
+                rdict['roi'] = self.resample_bin(roi, wshape_new, hshape_new, 0.0)
```

Comment:
I thought `roi_data` was only one element. Different from `gt_data` which is now a list.

Reply:
Yes you are totally right, but the thing is in medicaltorch a this line https://github.com/neuropoly/medicaltorch/blob/master/medicaltorch/datasets.py#L257, we treat roi_filename the same as gt_filename (line 260), so roi_filename need to be a list since I modified SegmentationPair2D to deal with list instead of a single file.

Reply:
Mmmmm so true! Perfect